public/scripts/main.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(() => {
+  function onKonamiCode(cb) {
+    let input = "";
+    const key = "38384040373937396665";
+
+    document.addEventListener("keydown", (e) => {
+      input += (`${e.keyCode}`);
+
+      if (input === key) {
+        return cb();
+      }
+
+      if (!key.indexOf(input)) { return 1 }
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/jdevuono54/OCR-P5/pullRequest?prid=8453331)
```suggestion
      if (!key.indexOf(input)) { return 1; }
```